### DISCUSSION TITLE
not receiving wLuna

### DISCUSSION THREAD

**Discussion by wwnromadhon** [wormhole-foundation/wormhole#232] [Problems / bugs] [answered]
i swap wLuna from solana to eth use wormhole but didn't receiving it.
here is my tx on solana :

https://explorer.solana.com/tx/2dmj3tsAGq6Wep5J9GMGHPTwTm1tpzHz1hNxN5DF4V5TGCoCD4yaMLAdAAECUgcjPs9Nx4Ti8Eq29TTFRFC8xvPk

can it send back to my solana address? if can't, please resolve that and here is my eth address:

0x37e83B3F94d635FC51598a4bfdF3522aB2C08DD8

thank you.

**Comment by hendrikhofstadt** [comment, marked as answer]
It looks like you didn't complete the transfer on the Ethereum side but the Wormhole correctly attested your transfer. Please follow the instructions of the UI you used. We don't provide support for UIs.

**Comment by leoluk** [comment]
You can create issues for wormholebridge.com here: https://github.com/solana-labs/oyster/issues

**Comment by leoluk** [comment]
Did this get resolved? Your tokens aren't lost, your wallet provider simply needs to retry the submission to Ethereum.
